lib/phoenix_live_view/channel.ex

```diff
@@ -564,6 +578,7 @@ defmodule Phoenix.LiveView.Channel do
         },
         action
       )
+      |> Utils.merge_flash(flash)
```

Comment:
Suggestion: because we need to consider the flash every time the socket is mounted, make it a parameter for `configure_socket`, like the action.